help_function/src/dnnl_utils_activation.cpp

```diff
@@ -89,7 +89,7 @@ void test1() {
     auto s = (handle.activation_forward(desc, alpha, dataTensor, data, beta,
                                         outTensor, out),
               0);
-
+    dev_ct1.queues_wait_and_throw();
```

Comment:
Please create a JIRA to followup the test update,  wait is not neccssary if oneDNN/compiler fix the syn issue for host_task in in-order sycl::queue in CPU device.